src/api/app/controllers/webui/package_controller.rb

```diff
@@ -792,12 +792,23 @@ def set_job_status
   end
```

Comment:
@DavidKang Please add a short explanation what was causing this to the commit message. Such additional information can be quite helpful when trying to understand why some changes have been made. Imagine some one else or you in three month has to debug something in this code.
Then a short explanation in the commit message can save you quite some time